### DISCUSSION TITLE
A "Deployment context" for actions

### DISCUSSION THREAD

**Discussion by AngellusMortis** [community/community#13125] [Actions]
Recently we have ran into some [issues](https://github.com/integrations/slack/issues/1364) and [limitations](https://github.com/github/feedback/discussions/13124) with the out of the box Slack Notifications for deployments, so we have been making our own. 

It would be great to have a "deployment" context or something similar to result of a deployment run for use for generating notifications. Something similar to:

```json
{"production": {
    "environment": "production",
    "url": "http://example.com",
    "needs_approval": true,
    "approved": true,
    "reviewer": "AngellusMortis"
    "review_text": "I approve this deploy",
}}
```

**Comment by ethomson** [comment]
Thanks for the feedback @AngellusMortis - I'll discuss this with the team working on slack integrations so that they can make sure that they this. Appreciate you taking the time to write this up, it's very helpful.
